recipes/warrant-lite/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "warrant-lite" %}
+{% set version = "1.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 16e9d6a2c9599f7a345877addbe2ed8203b76db6ded942cc5d6f77ab4ec6c49e
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
```

Comment:
Did you mean?
```suggestion
    - python >=3.6
```

Reply:
Ops, you are right, I misread it
Thanks! :)